docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -1116,6 +1133,28 @@ public Consumer<Message<String>> consume2() {
 }
 ----
 
+===== Resource Provision
+Service bus binder supports provisioning of queue, topic and subscription, users could use below properties to enable provisioning.
+[source,yaml]
+----
+spring:
+  cloud:
+    azure:
+      credential:
+        tenant-id: ${AZURE_TENANT_ID}
+      profile:
+        subscription-id: ${AZURE_SUBSCRIPTION_ID}
+      servicebus:
+        resource:
+          resource-group: ${AZURE_SERVICEBUS_RESOURECE_GROUP}
+    stream:
+      servicebus:
+        bindings:
+          servicebus-binding:
```

Comment:
here let's change the binding name from "servicebus-binding" to <binding-name> to help users understand the meaning of this property.